mlflow/pytorch/__init__.py

```diff
@@ -15,6 +15,7 @@
 import cloudpickle
 import numpy as np
 import pandas as pd
+from packaging.version import Version
```

Comment:
Can you remove packaging here as we discussed?

Reply:
yeah sure  i will replace it with LooseVersion and upload the latest code today.

Reply:
Updated the code.